aws-android-sdk-iot/build.gradle

```diff
@@ -12,14 +12,14 @@ android {
 }
 
 dependencies {
-    api (project(':aws-android-sdk-core')){
+    api (project(':aws-android-sdk-core')) {
         exclude group: 'com.google.android', module: 'android'
     }
-    implementation 'org.conscrypt:conscrypt-android:2.0.0'
-    implementation 'org.eclipse.paho:org.eclipse.paho.client.mqttv3:1.2.2'
+    implementation 'org.conscrypt:conscrypt-android:2.4.0'
+    implementation 'org.eclipse.paho:org.eclipse.paho.client.mqttv3:1.2.4'
```

Comment:
Didn't the paho update from 1.2.2 to 1.2.3 break something and we had to reverse?

Reply:
1.2.3 did break. 1.2.4 appears to be fixed again. I ran the IoT tests.